common/src/IO/GameConfigParser.cpp

```diff
@@ -39,17 +39,28 @@ namespace TrenchBroom {
         GameConfigParser::GameConfigParser(const std::string& str, const Path& path) :
         ConfigParserBase(str, path) {}
 
+        namespace {
+            void checkVersion(const EL::Value& version) {
+                const std::set<EL::IntegerType> validVsns({3, 4});
```

Comment:
I'd prefer `std::vector`, we really prefer `std::vector` unless it doesn't make sense.